chemclipse/plugins/org.eclipse.chemclipse.ux.extension.xxd.ui/src/org/eclipse/chemclipse/ux/extension/xxd/ui/swt/ExtendedWellChannelsUI.java

```diff
@@ -91,7 +88,6 @@ private void createToolbarMain(Composite parent) {
 		//
```

Comment:
When changing an element, the size needs to be adjusted if elements are added/removed:
`composite.setLayout(new GridLayout(2, false)); `